### PR TITLE
Update personnel contact information

### PR DESCRIPTION
**Pull request by vdelau**
This PR is meant to collect changes to personnel contact information, for instance to replace Twitter for Mastodon. Please comment or code review with additional changes to be made.

~~_I've based this change on master, so the Icicle/AMPHP rename is not yet in here. Once that is merged, I'll rebase this branch._~~

### CODE REVIEW

**Comment by Crell** [comment]
Please change me to 
Link: https://phpc.social/@Crell
Label: @Crell@phpc.social

**Comment by Jean85** [comment]
Same for me, `AlessandroLai` is my PHPC handle. Check for syntax though, the `@` is messing stuff up in markdown! :D

**Comment by alekitto** [comment]
Can you change me to:

```
@alekitto@phpc.social
https://phpc.social/@alekitto
```

Thanks

**Comment by BusterNeece** [comment]
Can update my contact info to:

```
@BusterNeece@phpc.social
https://phpc.social/@BusterNeece
```

**Comment by kenguest** [comment]
It would be great if you could update mine to
@kenguest@phpc.social
https://phpc.social/@kenguest

Thanks!

**Comment by vdelau** [comment]
Rebased from master, so that the Icicle/AMPHP rename is also in.

**Comment by ashnazg** [comment]
Same change for me:
@[ashnazg@phpc.social](mailto:ashnazg@phpc.social)
https://phpc.social/@ashnazg

**Comment by WyriHaximus** [comment]
Here is mine: 
```
@wyri@haxim.us
https://toot-toot.wyrihaxim.us/@wyri
```

**Comment by mbniebergall** [comment]
Update mine too please, thanks!

@mbniebergall@phpc.social
https://phpc.social/@mbniebergall

File: personnel.md [hunk 1]
```diff
@@ -4,74 +4,74 @@
...
+| Larry Garfield ([@Crell@phpc.social])             | 2016-12-24 - 2025-05-30 |
+| Matthew Weier O'Phinney ([@mwop])                 | 2016-12-24 - 2025-05-30 |
+| Cees-Jan Kiewiet ([@wyri@haxim.us])               | 2016-12-24 - 2024-08-31 |
+| Chris Tankersley ([@dragonmantank])               | 2016-12-24 - 2024-01-28 |
+| Korvin Szanto ([@korvinszanto])                   | 2016-12-24 - 2024-01-28 |
+| Chuck Burgess ([@ashnazg@phpc.social])            | 2018-08-26 - 2024-08-31 |
+| Enrico Zimuel ([@ezimuel])                        | 2020-01-25 - 2024-01-28 |
+| Alessandro Chitolina ([@alekitto@phpc.social])    | 2021-06-25 - 2025-05-30 |
+| Ken Guest ([@kenguest@phpc.social])               | 2022-01-30 - 2024-01-28 |
+| Jaap van Otterdijk ([@jaapio])                    | 2022-08-28 - 2024-08-28 |
```
Comment (jaapio) [inline personnel.md:29]:
```suggestion
| Jaap van Otterdijk ([@jaapio@phpc.social])                    | 2022-08-28 - 2024-08-28 |
```

File: personnel.md [hunk 2]
```diff
@@ -155,16 +155,17 @@ Feel free to contact the secretaries at info AT php-fig.org. For more informatio
 [@goetas_asmir]: https://twitter.com/goetas_asmir
 [@greydnls]: https://twitter.com/greydnls
 [@guilhermeblanco]: https://twitter.com/guilhermeblanco
+[@iansltx]: https://twitter.com/iansltx
 [@ieatkillerbees]: https://twitter.com/ieatkillerbees
+[@iminyaylo]: https://twitter.com/iminyaylo
+[@jaapio]: https://twitter.com/jaapio
```
Comment (jaapio) [inline personnel.md:161]:
```suggestion
[@jaapio@phpc.social]: https://phpc.social/@jaapio
```

**Review by Jean85 — APPROVED** [review]

**Pull request merged**